www/install.sh

```diff
@@ -93,14 +117,10 @@ while test $# -gt 0; do
   shift
 done
 
-command -v curl > /dev/null 2>&1 ||
-  command -v wget > /dev/null 2>&1 ||
-  err "need wget or curl (command not found)"
-
-need install
+need_one_of curl wget
```

Comment:
Doing `need unzip` or `need tar` in the `$target` case statement seems fine to me, instead of adding a `need_one_of` function.